config/config_helpers.go

```diff
@@ -208,6 +211,30 @@ func parseGetEnvParameters(parameters string) (EnvVar, error) {
 	return envVariable, nil
 }
 
+// runCommand is a helper function that runs a command and returns the stdout as the interporation
+// result
+func runCommand(parameters string, terragruntOptions *options.TerragruntOptions) (string, error) {
+	args := parseParamList(parameters)
+
+	if len(args) == 0 {
+		return "", errors.WithStackTrace(EmptyStringNotAllowed("parameter to the run_cmd function"))
+	}
+
+	currentPath := filepath.Dir(terragruntOptions.TerragruntConfigPath)
```

Comment:
The idea as that we run the command from the module folder because the command might need the path as the context. A good example (my actual use-case) is if you want to write a script that returns the bucket name based on the account (folder) where you are running the command.

Using the `WorkingDir` wouldn't work when you run *-all between accounts using `--terragrunt-include-dir`

Reply:
OK, fair enough. Please make sure to document that in the README!